csharp/src/SeedLang.Ast/AstNode.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Ast {
+  // The base class of all AST nodes.
+  public abstract class AstNode {
+    // Create the string representation of the AST node.
```

Comment:
Although it's not mentioned by Google's or Microsoft's C# style doc, most Google coding styles enforce a descriptive style for file/class/function comments. For example:

> Google python style: The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable."""). 

> Google JavaScript style: Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.

I think either way is okay but we should better stick to one of them. How about following Google's convention to use the third-person phrases for file/class/function/method comments?

Here and the rest.

Reply:
Done